apps/voting/app/src/components/VoteCard/VoteOptions.js

```diff
@@ -1,11 +1,28 @@
 import React from 'react'
+import BN from 'bn.js'
 import { Spring, config as springs } from 'react-spring'
-import VoteOption from './VoteOption'
+import { divideRoundBigInt } from '@aragon/ui'
 import { percentageList } from '../../math-utils'
+import VoteOption from './VoteOption'
 
 const ANIM_DELAY_MIN = 100
 const ANIM_DELAY_MAX = 800
 
+function optionPercentage(optionPower, totalVotingPower) {
+  const precision = 10 ** 9
+  const precisionBn = new BN(precision)
+
+  return (
+    parseInt(
+      divideRoundBigInt(
```

Comment:
It's starting to look like we may want another utility :).

Maybe `divideBN()` and we can give it a `decimalsPrecision` option?